Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.EntityFrameworkCore;
 using Promact.Trappist.Web.Models;
```

Comment:
If i remove EntityFrameworkCore then DbContextOptions,OnModelCreating,DbSet throwing error.
and if i remove web.Model then ApllicationUser is Throwing Error Messege